recipes/wget/meta.yaml

```diff
@@ -0,0 +1,29 @@
+package:
+  name: wget
```

Comment:
Should we rename to `gnu-wget`? Otherwise, we will have a name clash with the `wget` python library. Since the anaconda channel is top priority, we cannot shadow that package.